src/main/java/frc/robot/commands/StartShooterCommand.java

```diff
@@ -15,6 +15,6 @@
  */
 public class StartShooterCommand extends SequentialCommandGroup  {
     public StartShooterCommand(ShooterSubsystem shoot, ElevatorSubsystem elevator) {
-        addCommands(elevator.shiftBack(), shoot.turnOnShooter(), new DelayCommand(0.5), elevator.start());
+        addCommands(elevator.shiftBack(), shoot.turnOnShooter());
```

Comment:
see earlier feedback, a generic class that avoids the need for a separate class would be great here